Common/include/CConfig.hpp

```diff
@@ -1136,6 +1136,10 @@ class CConfig {
   nHistoryOutput, nVolumeOutput;  /*!< \brief Number of variables printed to the history file. */
   bool Multizone_Residual;        /*!< \brief Determines if memory should be allocated for the multizone residual. */
 
+  unsigned short n_scalars,       /*!< \brief Number of transported scalars. */
+  //n_species,                      /*!< \brief Number of species in multicomponent flow. */
```

Comment:
remove any commented code you don't need

Reply:
done